### PR TITLE
add UTs

### PR DESCRIPTION
**Pull request by Vladlis**

### CODE REVIEW

File: web-project/src/test/java/com/epam/webproject/service/AuthorServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    @InjectMocks
+    private AuthorService authorService;
+
+    @Before
+    public void init() {
+        MockitoAnnotations.initMocks(this);
+    }
+
+    @Test
+    public void testGetByName() {
```
Comment (sqshq) [inline web-project/src/test/java/com/epam/webproject/service/AuthorServiceTest.java:38]: I also suggest you to read about BDD practices like splitting the test into given/when/then parts, and shouldXXXwhenYYY test naming convention. You could practice on this particular method

**Review by sqshq — APPROVED** [review]

File: web-project/src/test/java/com/epam/webproject/service/AuthorServiceTest.java (hunk 1 above)
Comment (Vladlis) [reply web-project/src/test/java/com/epam/webproject/service/AuthorServiceTest.java:38]: @sqshq,
Updated the test. Is that what you meant?
And is it useful for such simple tests? Looks like method naming of testing frameworks make it obvious (when, assertEquals).

Comment (sqshq) [reply web-project/src/test/java/com/epam/webproject/service/AuthorServiceTest.java:38]: Yes, it is useful for any kind of tests. And that's not what I meant actually - //when part should contain the action itself, `authorService.getByName(...)` in this case.

Comment (Vladlis) [reply web-project/src/test/java/com/epam/webproject/service/AuthorServiceTest.java:38]: Done

Comment (Vladlis) [reply web-project/src/test/java/com/epam/webproject/service/AuthorServiceTest.java:38]: @sqshq
Ok now?

Comment (sqshq) [reply web-project/src/test/java/com/epam/webproject/service/AuthorServiceTest.java:38]: Yes

**Pull request merged**